SwaprHQ/swapr-dapp
# [SWA-138][DOCS] - Update docs url

## Fixes: [SWA-138](https://linear.app/swaprhq/issue/SWA-138/update-docs-url)

# Description
* Update all the references to the old DXD docs, to use now the new Gitbook Pages from Swapr.


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr dapp page
4) Links in the footer to the docs and to the Brand Kit should point to the new Gitbook pages from Swapr.
